demo/src/main/resources/frontend/angular-app/src/app/offers/offers.component.html

```diff
@@ -0,0 +1,41 @@
+<section>
+  <table>
+    <thead>
+    <tr>
+      <!--<th>Kontynent</th>
+      <th>Kraj</th>-->
+    </tr>
+    </thead>
+    <tbody>
+    <tr *ngFor="let offer of offers">
+      <td>{{offer.continent}}</td>
+      <td>{{offer.country}}</td>
+      <td>
+        <a routerLink="/edit/{{offer.id}}" class="button button-northAmerica">Ameryka Północna</a>
```

Comment:
Tutaj, obojętnie który kraj wybierzesz zawsze przejdzie na ten sam ekran. Jeżeli to niedokończona wersja to ok, inaczej trzeba by to było zmienić